experiments/yarn/src/main/scala/org/apache/gearpump/experiments/yarn/master/ResourceManagerCallbackHandler.scala

```diff
@@ -60,19 +76,19 @@ class ResourceManagerCallbackHandler(appConfig: AppConfig, am: ActorRef) extends
 
     //todo:wont work with non uniform containers
     (0 until requestCount).foreach(request => {
-      am ! ContainerRequestMessage(containerCount, appConfig.getEnv(GEARPUMPMASTER_VCORES).toInt)
+      resourceManagerClient ! AdditionalContainersRequest(containerCount)
```

Comment:
First, the getProgress means whether the applicaiton is finished, instead of whether the container is allocated.

As Gearpump is long running, so  the progress should always be smaller than < 100%.
I think we can adopt a very simple strategy here, we always  set the progress to 50%, as the YarnAppMaster is already started, and it is long running.